BUILD.ndk_clang.tpl

```diff
@@ -59,7 +59,7 @@ cc_toolchain_suite(
 [ndk_cc_toolchain_config_rule(
     name = "toolchain_config_%s" % target_system_name,
     api_level = {api_level},
-    clang_resource_directory = "{clang_resource_directory}",
+    clang_resource_directory = glob(["lib64/clang/*"], exclude_directories = 0, allow_empty = False)[0],
```

Comment:
This will work, though if for some reason some future version of the NDK has multiple versions, there's no telling which entry you're going to get (and some might find using globs in this way slightly surprising). I was thinking to do something more explicit in the repository rule implementation in `rules.bzl`, something like:

```
lib64_clang_directory = clang_directory + "/lib64/clang"
lib64_clang_files = ctx.path(lib64_clang_directory).readdir()
if len(lib64_clang_files) != 1:
  fail("Expected 1 directory under " + lib64_clang_directory)
clang_resource_directory = lib64_clang_files[0]
```

(i haven't tested that)